tests/unit/test.program.js

```diff
@@ -329,15 +331,27 @@ describe('program.main', () => {
 
 
 describe('program.defaultVersionGetter', () => {
-
-  it('returns the package version', () => {
-    let root = path.join(__dirname, '..', '..');
+  let root = path.join(__dirname, '..', '..');
+  let testProcess = {
```

Comment:
This shouldn't be shared be a shared variable because it makes the tests unpredictable (say, if they were run in reverse order). I'd say keep it defined as a local constant within each respective test function.